File: src/Day2.java
import java.util.Scanner;

public class Day2 {
    // Complete the solve function below.
    static void solve(double meal_cost, double tip_percent, double tax_percent) {
        int cost = (int) Math.round(meal_cost * (1 + tip_percent / 100 + tax_percent / 100));
        System.out.println(cost);
    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        double meal_cost = scanner.nextDouble();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        int tip_percent = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        int tax_percent = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        solve(meal_cost, tip_percent, tax_percent);

        scanner.close();
    }
}

File: python/day_2.py
import math
import os
import random
import re
import sys

# Complete the solve function below.
def solve(meal_cost, tip_percent, tax_percent):
    cost = meal_cost * (1 + tip_percent / 100 + tax_percent / 100)
    print(round(cost))

if __name__ == '__main__':
    meal_cost = float(input())

    tip_percent = int(input())

    tax_percent = int(input())

    solve(meal_cost, tip_percent, tax_percent)
